src/pages/Profile.js

```diff
@@ -1,21 +1,23 @@
 import React, { Component } from 'react';
-import {AsyncStorage,StyleSheet,Text, View,Image,TouchableOpacity } from 'react-native';
+import firebase from 'firebase';
+import {StyleSheet,Text, View,Image,TouchableOpacity } from 'react-native';
```

Comment:
add spaces between components

```suggestion
import { StyleSheet, Text, View, Image, TouchableOpacity } from 'react-native';
```